tests/testthat/test-derive_params_growth_age.R

```diff
@@ -4,18 +4,18 @@
 test_that("derive_params_growth_age Test 1: derive_params_growth_age works", {
   vs_data <- tibble::tribble(
     ~USUBJID, ~SEX, ~AGECUR, ~AGEU, ~VSTESTCD, ~VSSTRESN,
-    "1001", "M", 30, "days", "WEIGHT", 10,
-    "1002", "F", 25, "months", "WEIGHT", 100,
+    "1001", "F", 24.5, "months", "WEIGHT", 10,
+    "1002", "F", 25.49, "months", "WEIGHT", 11,
```

Comment:
demo'd here with fake metadata but showcases that it does indeed work

Reply:
can we add a similar extra unit test for the by height with height values like 0.11, 0.15, 0.19 to show that these respectively end up using the 0.1, 0.1, 0.2 metadata rows? saves Sukalpo adding this then in his open PR